lametro/events.py

```diff
@@ -43,7 +46,7 @@ def _find_partner(self, event):
             return partner
 
         elif event.is_spanish:
-            raise ValueError("Can't find English companion for Spanish Event {}".format(event['EventId']))
+            logger.warning("Can't find English companion for Spanish Event {}".format(event['EventId']))
```

Comment:
let's have this continue to break – we should never accept an unpaired spanish event.